python/lsst/ctrl/bps/parsl/configuration.py

```diff
@@ -118,6 +118,9 @@ def get_workflow_name(config: BpsConfig) -> str:
     name : `str`
         Workflow name.
     """
+    workflow_name = get_bps_config_value(config, "workflow_name", str, "")
```

Comment:
I think it makes it clearer to leave off the default.

But I would rather use something that BPS has already defined (like `uniqProcName`), rather than inventing a new configuration value. I think `<project>.<campaign>` works well. Why do you think that needs to change?

This change would also need to be documented in the function docstring and the README.

Reply:
I removed this change in a later commit.